### PR TITLE
[feat] Add and Style Images to Project Modal

### PR DESCRIPTION
**Pull request by nehaahussain**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Renamed query to `queryProjectbyId` 
- Styled and added the project image (for the specific project id) to the project modal.

### Screenshots
Screenshot 2024-10-19 at 6 25 10 PM

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

- In app/page.tsx, change the `project_id` on line 23 to`2` instead of `1` for BIBBLE!
- ex: `<ProjectModal project_id={2}></ProjectModal>`
- run `pnpm dev` and press the open button on top of the google maps

CC: @itsliterallymonique

### CODE REVIEW

File: pnpm-lock.yaml
Comment (itsliterallymonique) [inline pnpm-lock.yaml:14]: Don't merge this file to main

File: package.json
Comment (itsliterallymonique) [inline package.json:15]: Don't merge this file into main

**Review by itsliterallymonique — COMMENTED** [review]
Great work Neha! Just remove package.json and pnpm-lock.yaml from the commit since you didn't install any new packages. Also could you make the PR screenshot uses the non-bibble image (In project_id 1).

**Comment by nehaahussain** [comment]
@itsliterallymonique Updated!

**Comment by itsliterallymonique** [comment]
> @itsliterallymonique Updated!

Perfect, thanks Neha! You can squash and merge once Julee approves!

**Review by jjstnlee — APPROVED** [review]
Looks good to me :D

File: components/ProjectModal/index.tsx
Comment (nehaahussain) [reply components/ProjectModal/index.tsx:3]: idk if its the same for yall but i got some jsx errors if i didnt add that

**Pull request merged**
